packages/web-react/src/types/shared/dictionaries.ts

```diff
@@ -58,3 +58,12 @@ export const ValidationStates = {
 
 export type ValidationStatesKeys = keyof typeof ValidationStates;
 export type ValidationStatesTypes<T> = (typeof ValidationStates)[ValidationStatesKeys] | T;
+
+export const Align = {
```

Comment:
I am not sure about this naming. `align` seems to me too general and I can imagine situations where vertical and horizontal aligns should be used. In this case of the dictionary I would suggest using `HorizontalAlignment` instead. But we should have some concept about alignment.

Reply:
After channel discussion, we agreed with `AlignmentX`, so i renamed to it.